README.md

```diff
@@ -141,6 +141,11 @@ Firecracker currently only supports little-endian platforms. Firecracker will
 not compile for big-endian architectures, and will not work correctly with
 big-endian configured guests.
 
+## Supported kernels
```

Comment:
I would name the version explicitly and then link for more details our support policy.